examples/flutter_gallery/lib/demo/material/overscroll_demo.dart

```diff
@@ -32,9 +32,7 @@ class OverscrollDemoState extends State<OverscrollDemo> {
          content: new Text("Refresh complete"),
          action: new SnackBarAction(
            label: 'RETRY',
-           onPressed: () {
-             _refreshIndicatorKey.currentState.show();
-           }
+           onPressed: _refreshIndicatorKey.currentState.show
```

Comment:
This is not equivalent. What if `_refreshIndicatorKey.currentState` has changed value before the user taps the button?